File: gateOne/day five/java/TaskOne.java
import java.util.Scanner;
public class TaskOne{
public static void main(String[] args){
	validate();

}
	public static void  validate(){
	Scanner scanner = new Scanner(System.in);
		String userInput;
		 do{
			System.out.print("Enter card number you want to validate : ");
				String user = scanner.next(); 
				userInput = user.replace("-", "");
		}while(userInput.length() < 16 || userInput.length() > 16);

 
		if (userInput.charAt(0)== '4' || userInput.charAt(0)== '5' || userInput.charAt(0)== '6'){
			System.out.print(userInput.charAt(0));
			System.out.print(userInput.charAt(1));
			System.out.print(userInput.charAt(2));
			System.out.print(userInput.charAt(3)+"-");
			System.out.print(userInput.charAt(4));
			System.out.print(userInput.charAt(5));
			System.out.print(userInput.charAt(6));
			System.out.print(userInput.charAt(7)+"-");
			System.out.print(userInput.charAt(8));
			System.out.print(userInput.charAt(9));
			System.out.print(userInput.charAt(10));
			System.out.print(userInput.charAt(11)+"-");
			System.out.print(userInput.charAt(12));
			System.out.print(userInput.charAt(13));
			System.out.print(userInput.charAt(14));
			System.out.print(userInput.charAt(15)+".");


			System.out.print("\nYour credit card is valid");
		}else{
			System.out.print("Your credit card is invalid");
			}



	}
}

File: gateOne/day five/phyton/task_one.py
def validator():
	user_input = " "
	while len(user_input) != 16:
		user = input("Enter the card number u want to validate : ")
		user_input  = user.replace("-","")
	if user_input[0]== '4' or user_input[0]== '5' or user_input[0]== '6':
			print(user_input[0],end="")
			print(user_input[1],end="")
			print(user_input[2],end="")
			print(user_input[3]+"-",end="")
			print(user_input[4],end="")
			print(user_input[5],end="")
			print(user_input[6],end="")
			print(user_input[7]+"-",end="")
			print(user_input[8],end="")
			print(user_input[9],end="")
			print(user_input[10],end="")
			print(user_input[11]+"-",end="")
			print(user_input[12],end="")
			print(user_input[13],end="")
			print(user_input[14],end="")
			print(user_input[15]+".",end="")
			print("\nYour credit card is valid")
	else:
		print("Your card is invalid")
validator()